audinterface/core/utils.py

```diff
@@ -123,7 +123,19 @@ def read_audio(
         signal: array with signal values in shape ``(channels, samples)``
         sampling_rate: sampling rate in Hz
 
-    """
+    Example:
+        >>> import audb
+        >>> media = audb.load_media(
+        ...     'emodb',
+        ...     'wav/03a01Fa.wav',
+        ...     version='1.2.0',
+        ...     verbose=False,
+        ... )
+        >>> signal, sampling_rate = read_audio(media[0], end=pd.Timedelta(0.01, unit='s'))
```

Comment:
I completely forgot about this function, it's actually nice to have because I find myself converttng `start` and `end` of an index to `offset` and `duration` to use it with `audiofile.read()`.